src/client/components/TotalPriceCard/TotalPriceCard.styles.css

```
@@ -1,10 +1,9 @@
-.wrapper {
+.wrapperTotalPrice {
```

```suggestion
.wrapper-total-price {
```

Use this other syntax instead. Apply the same to the rest of the names

---

I have used kebab-case as per your suggestion on all the css elements 👍